### DISCUSSION TITLE
Adding a scrollbar

### DISCUSSION THREAD

**Discussion by Retropunchout** [peter-kish/gloot#152] [Q&A]
Hello,
I'm trying to add a scrollbar for my CtrlInventoryGridEx inventory, and struggling how best to do this? Using a ScrollContainer does work, but if the dragged item goes outside it it is invisible.

Apologies in advance if I've just messed up with zlevels/visibility/layering.

**Comment by peter-kish** [comment]
Hi,
Yeah, that one's a known issue caused by a somewhat "hacky" way of doing drag&drop in the plugin (mostly because Godot's default drag&drop mechanism is not very configurable).

But I think I already have a fix ready for the next release (v2.4.1).

**Comment by peter-kish** [comment]
This should be fixed in [v2.4.1](https://github.com/peter-kish/gloot/releases/tag/v2.4.1).
